src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_PointerElementAccess.cs

```diff
@@ -71,7 +73,7 @@ private BoundExpression RewritePointerElementAccess(BoundPointerElementAccess no
                     additionKind,
                     rewrittenExpression,
                     rewrittenIndex,
-                    rewrittenExpression.Type,
+                    rewrittenExpression.Type!,
```

Comment:
I'm surprised we would need to suppress twice.

Reply:
We dereference rewrittenIndex.Type above, not rewrittenExpression.Type.

---
In reply to: [387364421](https://github.com/dotnet/roslyn/pull/42068#discussion_r387364421) [](ancestors = 387364421)